app/lib/reportback/test/reportback.test.js

```diff
@@ -6,12 +6,15 @@ var assert = require('assert')
   ;
 
 function test() {
-  var TEST_PHONE = '15555555555';
-  var TEST_CAMPAIGN_CONFIG = app.getConfig(app.ConfigName.REPORTBACK, 'test', 'endpoint');
+  var TEST_PHONE = '15555555555'
+    , TEST_CAMPAIGN_CONFIG = app.getConfig(app.ConfigName.REPORTBACK, 'test', 'endpoint')
+    , testDoc // exists in the global test scope; variable re-assigned each time createTestDoc() is run. 
```

Comment:
Instead of pulling this var up to the global scope, I think I'd prefer we continue using the `testDoc` vars in their previously more local scope and just add the campaign property. **Or** to pass in `testData` to both parameters - ex: `reportback.receivePhoto(testData, testData);`

Having this particular mutable variable in the global scope can be confusing without context of the entire setup of tests in this file. Like, I can see `before(createTestDoc);` in a test, but not necessarily know intuitively that the `testDoc` var has now just been changed.